### DISCUSSION TITLE
Controle de dependências para grupos de projetos

### DISCUSSION THREAD

**Discussion by fardeadbr** [HashLoad/boss#74] [Q&A] [answered]
Atualmente tenho uma pasta com a seguinte estrutura:

```
> api_console/ (pasta com dpr)
> api_winsrc/ (pasta com dpr)
> src/

> modules/

> projeto.groupproj
> boss.json
```

Trata-se de um **grupo de projeto no Delphi** que contém **duas API em Horse**, compartilhando o mesmo src. Uma sendo modo Console e outro serviço.

No **boss.json** eu inclui os dois projetos: 

```
[...]
"projects": [
	"./apiconsole/LMAuth.Console.dproj",
	"./apiwinsrc/LMAuth.WinSrc.dproj"
],
[...]
```

Ao realizar o comando: `boss install horse`
Ele pede para recarregar (nos dois projetos), porém ao incluir o path esta com o caminho inválido. **Como se a pasta modules estivesse na raiz**.

Atualmente ele adiciona: **`module/horse/src`**

Quando na verdade deveria ser: **`../module/horse/src`**

Devido a pasta do projeto esta em um outro nível `./api_console` e `./api_winsrc`

![paths](https://user-images.githubusercontent.com/33734510/137573238-d6d5fcfd-592e-4198-b9ae-117358d98615.png)
![boss-json](https://user-images.githubusercontent.com/33734510/137573246-12d25887-9407-4477-8a99-348143eea705.jpg)
![delphi](https://user-images.githubusercontent.com/33734510/137573437-1203f4d6-fbfd-4847-b49b-f145e3de4b0b.png)

É possível configurar isso?

**Comment by viniciussanchez** [comment, marked as answer]
Atualmente não é possível, mas podemos ver para a próxima versão
